main.py

```diff
@@ -0,0 +1,30 @@
+from arguments import parameters
+from keys import check_keys
+from requester import request_weatherapi, request_openmeteo, request_meteostat
+
+# get parameters
+param = parameters()
+print(param)
```

Comment:
Even if you're using just a regular print here, please at least notify the user that these are the arguments that will be used by the requester, or sth like that. I can see that you do such a thing for e.g. the keys, so please be as consistent and user-friendly as you can;) This applies to all such print-something parts here.

Reply:
done